src/repr/src/row.rs

```diff
@@ -187,8 +187,14 @@ enum Tag {
     Timestamp,
     TimestampTz,
     Interval,
-    Bytes,
-    String,
+    BytesTiny,
+    BytesShort,
+    BytesLong,
+    BytesHuge,
+    StringTiny,
+    StringShort,
+    StringLong,
+    StringHuge,
```

Comment:
not sure, but would `Bytes8`, `Bytes16`, ... possibly be clearer?

Reply:
Was going to say that too! +1, or however the kids say it.

Reply:
I considered that, but decided against it because it makes it sound like a statically-size field that holds up to 8 (resp. 16, 32, 64) bytes.

I also considered Bytes255, Bytes65535, etc., but that looks horribly ugly